go.mod

```diff
@@ -14,3 +14,24 @@ require (
 	github.com/vektra/mockery/v2 v2.10.0
 	golang.org/x/tools v0.1.8
 )
+
+require (
```

Comment:
why is there second require?
also not sure why all these libraries got pull in here?